### PR TITLE
kscript: Add version 4.1.1

### PR DESCRIPTION
**Pull request by Dany-coder778**
Added kscript.json to the bucket 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by goto1134** [comment]
There should be a dependency on kotlin, as the [installation guide](https://github.com/kscripting/kscript#installation) says

**Comment by goto1134** [comment]
shouldn't it be `depends` instead of `suggest`?

**Comment by Dany-coder778** [comment]
Maybe the user already has a kotlin compiler in their PATH and doesn't want another one.  It would still be fine for me to put it as a dependency, but maybe some people might not like having kotlin installed as well.  I also noticed that many jsons only suggest installing the visual c redistributables and don't enforce it as a dependency.

**Comment by goto1134** [comment]
It is an interesting question: how should it work in scoop? Looking and macOS brew, it force installs the dependencies.

But brew is de facto standard macOS tool. We have so many alternative package managers on windows... It might be reasonable not to install the dependencies.

Maybe we need maintainers to shed some light on the desired behavior?

**Comment by goto1134** [comment]
I looked up the documentation, and it says:

> ... when you install an app, Scoop will assume you want to install anything required that you don't already have and install these automatically too.

So that means kotlin should be in `depends`, I guess.

**Comment by Dany-coder778** [comment]
I replaced suggest with depends and added a note informing the user of installing kotlin.

File: bucket/kscript.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "url": "https://github.com/kscripting/kscript/releases/download/v4.1.1/kscript-4.1.1-bin.zip",
+    "hash": "127d169f5dc0f67800b99552c0db0e8b662de4dbe1ff8f2078a7c1cb9a5ed5c1",
+    "extract_dir": "kscript-4.1.1", 
+    "bin": "bin\\kscript.bat",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/kscripting/kscript/releases/download/v$version/kscript-$version-bin.zip",
+        "extract_dir": "kscript-$version"
+    }
+}
```
Comment (goto1134) [inline bucket/kscript.json:17]:
```suggestion
}

```

**Comment by goto1134** [comment]
@Dany-coder778, there is an error
@HUMORCE, can you please review this PR? It has been stale for a lot of time but contains a pretty useful piece of software

File: bucket/kscript.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "4.1.1",
+    "description": "Enhanced scripting support for Kotlin.",
+    "homepage": "https://github.com/kscripting/kscript",
+    "license": "MIT",
+    "notes": "This program will also install kotlin via Scoop.",
+    "depends": "kotlin",
```
Comment (HUMORCE) [inline bucket/kscript.json:7]:
```suggestion
    "suggest": {
        "Kotlin": "main/kotlin"
    },
```

**Pull request merged**